src/vector_store/__init__.py

```
@@ -0,0 +1,234 @@
+            observed: Name of the observed peer (document only)
+
+        Returns:
+            Namespace string in format:
+            - document: {prefix}.{workspace}.{observer}.{observed}
+            - message: {prefix}.{workspace}.messages
+        """
+        if namespace_type == "document":
+            if observer is None or observed is None:
+                raise ValueError(
+                    "observer and observed are required for document namespaces"
+                )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)